### ISSUE TITLE
Masks and other jquery interactions with inputs don't work inside wizard

### ISSUE THREAD

**Issue by strikegamer** [rstaib/jquery-steps#63]
I've found an issue when I tried to use some scripts in input texts inside the Wizard div.

When I try to interact with textboxes and jquery functions, like a Datepickers, maskedits, autocompletes, these scripts don't work inside the wizard area. It's not cause any errors, but simply don't work.

Checking the jquery.steps.js code, I tought I've found the cause.

In this part of code:
 // Transform the wizard wrapper and remove the inner HTML
    wizard.attr("role", "application").empty().append(stepsWrapper).append(contentWrapper)
        .addClass(options.cssClass + " " + options.clearFixCssClass + verticalCssClass);

the Wrappers are removed and after appended again. I've removed the empty() command to check, and created two forms.

Other test I made, was add the script to the textbox after this line. So, It works fine. But I added part of my code to do it, and it's not a good practice.

So, basically, on the page, outside wizard, all of scripts and textboxes works very well. Inside the wizard, all of then stops working.

I've created a Fiddler with an example:
http://jsfiddle.net/strikegamer/3bWm3/

Some idea how I can fix this? Or is a real bug?

**Comment by shanakaperera** [comment]
@rstaib I tried as you've mentioned in the StackOverflow answer.  First jQuery steps then the  [google places-autocomplete](https://developers.google.com/maps/documentation/javascript/examples/places-autocomplete) but it simply does nothing. But when the text-field is out of the wizard it works fine.
Any help or suggestion regarding this issue ??

**Comment by Burshyn** [comment]
First you need to initiate the steps and then execute the code.

**Comment by Itsmaqsud** [comment]
> [Here](http://jsfiddle.net/3bWm3/2/) you find the fixed example on JsFiddler.
Thanks!!!
